tensorflow/core/kernels/mkl_conv_grad_input_ops.cc

```diff
@@ -106,60 +111,79 @@ class MklConvBwdInputPrimitive : public MklPrimitive {
     context_.diff_dst_mem->set_data_handle(
         static_cast<T*>(const_cast<T*>(diff_dst_data)));
 
+#ifdef ENABLE_MKLDNN_V1
+    DCHECK_EQ(context_.bwd_input_primitives.size(),
+              context_.bwd_input_primitives_args.size());
+    for (size_t i = 0; i < context_.bwd_input_primitives.size(); ++i) {
+      context_.bwd_input_primitives.at(i).execute(
+          *context_.bwd_input_stream, context_.bwd_input_primitives_args.at(i));
+    }
```

Comment:
Please call `execute_primitives` for this.

Reply:
I somehow missed it. Fixed it.